sunpy/data/test/__init__.py

```diff
@@ -51,7 +55,7 @@ def get_test_filepath(filename, **kwargs):
     if isinstance(filename, Path):
         # NOTE: get_pkg_data_filename does not accept Path objects
         filename = filename.as_posix()
-    return get_pkg_data_filename(filename, package="sunpy.data.test", **kwargs)
+    return get_pkg_data_filename(filename, package=package, **kwargs)
```

Comment:
I wanted to use this function, but could not override package.

Either this is used inside sunpy only and we make all other packages call get_pkg_data_filename or this change needs to happen.

Reply:
Uh I mean fine, but why not just use `get_pkg_data_filename` directly from astropy?

Reply:
We could and I can close this PR.